FORMAT.md

```diff
@@ -118,29 +118,43 @@ The bracket element refers to a rigid body that connects modules, such as a ligh
   - R8HeavyRightInside
   - R8HeavyRightOutside  
 
+Currently, all of the given bracket elements have a single output interface.
+
 ### `<rigid-body>`
 
-The rigid body refers to a solid body with mass and one output.
+The rigid body refers to a solid body with mass and one or more outputs. Defaults to a single output interface. The rigid-body element can contain any number of `output` child elements, but no other child elements.
 
 **Required attributes:**
 `mass` (floating point formula, kg)
 
 **Optional attributes:**
-- `com_rot` (rotation matrix) the orientation of the center of mass (used for simplifying the inertia tensor description if desired).  Defaults to identity.
-- `com_trans` (translation vector, m) The position of the center of mass.  Defaults to (0,0,0).
-- `output_rot` (rotation matrix): the orientation of the output frame.  Defaults to identity.
-- `output_trans` (translation vector, m): The position the output frame.  Defaults to (0,0,0).
-- `ixx`, `iyy`, `izz`, `ixy`, `ixz`, `iyz` (floating point formulae, kg m^2) The 6 elements of the inertia tensor, relative to the COM frame as given above.  Each defaults to 0 (note, this means overall default is a point mass).
+- `com_rot` (rotation matrix): the orientation of the center of mass (used for simplifying the inertia tensor description if desired). Defaults to identity.
+- `com_trans` (translation vector, m): The position of the center of mass. Defaults to (0,0,0).
+- `output_rot` (rotation matrix): The default orientation of the output frames. Defaults to an identity matrix.
+- `output_trans` (translation vector, m): The default position the output frames. Defaults to (0,0,0).
+- `ixx`, `iyy`, `izz`, `ixy`, `ixz`, `iyz` (floating point formulae, kg m^2): The 6 elements of the inertia tensor, relative to the COM frame as given above.  Each defaults to 0 (note, this means overall default is a point mass).
 
-**Example:**
+**Examples:**
+
+Single-output rigid body:
 
 ```xml
 <rigid-body mass="0.5" com_trans="0.25 0 0" output_rot="Rx(pi/4)" output_trans="0.5 0 0"/>
```

Comment:
Given the definition, there is no way to specify a "zero output rigid body".  I thought about adding an explicit "number of outputs" attribute here, but couldn't really justify this.  I think defaulting to a single output is fine here; the only "downside" is that you have a frame in your kinematics for the output here.  If you really want this to be the end of a chain, you can always specify it as an end effector instead (although that has other implications)

Reply:
What about allowing the user to omit all the output related attributes
```xml
<rigid-body mass="0.5"/>
```
Require that they all be present or absent to reduce the risk of someone accidentally forgetting one?

Reply:
Right now that results in an identity output, so the same result twice. It should be fine if we omitt that?

Reply:
So, right now `<rigid-body mass="0.5"/>` is allowable, and results in a rigid body with a single identity output.  Without making a breaking version format change, we can't change the number of output frames this case produces; it should still result in a single output.

That said, I would be fine with disallowing the "output_rot" and "output_trans" `rigid-body` attributes when any `output` elements were present if you think that's more clear, and enforcing any non-identity values to be defined in the individual `output` elements.  The default value fallback to the base rigid body element sort of makes sense in my head, but I'm not strongly attached to it and don't think it would save a significant amount of typing.